src/containerapp/azext_containerapp/_params.py

```diff
@@ -249,6 +249,12 @@ def load_arguments(self, _):
         c.argument('target_port', type=int, validator=validate_target_port, help="The application port used for ingress traffic.")
         c.argument('exposed_port', type=int, help="Additional exposed port. Only supported by tcp transport protocol. Must be unique per environment if the app ingress is external.")
 
+    with self.argument_context('containerapp ingress ip-restriction') as c:
+        c.argument('allow_access', help='Boolean indicating whether the ip security restriction allows or denies access.')
```

Comment:
Could we define this parameter as `action='store_true'` ?

Reply:
This parameter changes frequently, and we wouldn't want the default behavior to change (i.e. customers might create an ip restriction that allows when they want to block an ip range)

Reply:
If the design of this parameter is not stable and has not been finalized, please add the `is_preview` flag to it.